src/utils.cpp

```diff
@@ -90,198 +82,200 @@ double pyne::endftod_cpp(char * s) {
-      dbl_exp *= (exp >>= 1) & 01? 1.0e8: 1;
+      dbl_exp = exp & 01 ? 10. : 1;
+      dbl_exp *= (exp >>= 1) & 01 ? 100. : 1;
+      dbl_exp *= (exp >>= 1) & 01 ? 1.0e4 : 1;
+      dbl_exp *= (exp >>= 1) & 01 ? 1.0e8 : 1;
       // Adjust for powers of ten from treating mantissa as an integer.
-      dbl_exp = (s[9] == '-'? 1/dbl_exp: dbl_exp) * 1.0e-6;
+      dbl_exp = (s[9] == '-' ? 1 / dbl_exp : dbl_exp) * 1.0e-6;
       // Get mantissa sign, apply exponent.
-      v = mant * (s[0] == '-'? -1: 1) * dbl_exp;
-    }
-    else {
-      mant = s[7] + 10 * s[6] + 100 * s[5] + 1000 * s[4] + 10000 * s[3] + \
+      v = mant * (s[0] == '-' ? -1 : 1) * dbl_exp;
+    } else {
+      mant = s[7] + 10 * s[6] + 100 * s[5] + 1000 * s[4] + 10000 * s[3] +
              100000 * s[1] - 111111 * '0';
       exp = s[10] + 10 * s[9] - 11 * '0';
-      dbl_exp = exp & 01? 10.: 1;
-      dbl_exp *= (exp >>= 1) & 01? 100.: 1;
-      dbl_exp *= (exp >>= 1) & 01? 1.0e4: 1;
-      dbl_exp *= (exp >>= 1) & 01? 1.0e8: 1;
-      dbl_exp *= (exp >>= 1) & 01? 1.0e16: 1;
-      dbl_exp *= (exp >>= 1) & 01? 1.0e32: 1;
-      dbl_exp *= (exp >>= 1) & 01? 1.0e64: 1;
-      dbl_exp = (s[8] == '-'? 1/dbl_exp: dbl_exp) * 1.0e-5;
-      v = mant * (s[0] == '-'? -1: 1) * dbl_exp;
+      dbl_exp = exp & 01 ? 10. : 1;
+      dbl_exp *= (exp >>= 1) & 01 ? 100. : 1;
+      dbl_exp *= (exp >>= 1) & 01 ? 1.0e4 : 1;
+      dbl_exp *= (exp >>= 1) & 01 ? 1.0e8 : 1;
+      dbl_exp *= (exp >>= 1) & 01 ? 1.0e16 : 1;
+      dbl_exp *= (exp >>= 1) & 01 ? 1.0e32 : 1;
+      dbl_exp *= (exp >>= 1) & 01 ? 1.0e64 : 1;
+      dbl_exp = (s[8] == '-' ? 1 / dbl_exp : dbl_exp) * 1.0e-5;
+      v = mant * (s[0] == '-' ? -1 : 1) * dbl_exp;
     }
   }
 
   // Convert an ENDF int to float; we start from the last char in the field and
   // move forward until we hit a non-digit.
   else {
     v = 0;
-    mant = 1; // Here we use mant for the place value about to be read in.
+    mant = 1;  // Here we use mant for the place value about to be read in.
     pos = 10;
     while (s[pos] != '-' and s[pos] != '+' and s[pos] != ' ' and pos > 0) {
       v += mant * (s[pos] - '0');
       mant *= 10;
       pos--;
     }
-    v *= (s[pos] == '-'? -1: 1);
+    v *= (s[pos] == '-' ? -1 : 1);
   }
   return v;
 }
 
-double pyne::endftod_f(char * s) {
+double pyne::endftod_f(char *s) {
 #ifdef PYNE_IS_AMALGAMATED
   return endftod_cpp(s);
 #else
   return endftod_(s, 12);
 #endif
 }
 
-double (*pyne::endftod)(char * s) = &pyne::endftod_f;
+double (*pyne::endftod)(char *s) = &pyne::endftod_f;
 
-void pyne::use_fast_endftod() {
-  pyne::endftod = &pyne::endftod_cpp;
-}
+void pyne::use_fast_endftod() { pyne::endftod = &pyne::endftod_cpp; }
 
 std::string pyne::to_upper(std::string s) {
   // change each element of the string to upper case.
-  for(unsigned int i = 0; i < s.length(); i++)
-    s[i] = toupper(s[i]);
+  for (unsigned int i = 0; i < s.length(); i++) s[i] = toupper(s[i]);
```

Comment:
There are a bunch of style changes here.  Does the style guide have an opinion, or are they arbitrary.  I'd rather not mix these changes in to the PR.

Reply:
old habits sorry....